ntbs-service/Migrations/20191121120513_AddAlertClassAndTestDerivedClass.cs

```diff
@@ -18,13 +18,10 @@ protected override void Up(MigrationBuilder migrationBuilder)
                     CreationDate = table.Column<DateTime>(nullable: false),
                     TbServiceCode = table.Column<string>(maxLength: 16, nullable: true),
                     CaseManagerEmail = table.Column<string>(maxLength: 64, nullable: true),
-                    HospitalId = table.Column<Guid>(nullable: true),

```

Comment:
Are we no longer including hospital at all?

Reply:
Nope - a comment nancy made on the ticket brought it to my attention that it wasn't just hospital ordering that was wrong on designs but also just having hospital in general